.github/workflows/ci.yml

```diff
@@ -54,46 +54,44 @@ jobs:
     runs-on: ubuntu-latest
     needs: [check]
     strategy:
+      fail-fast: false
       matrix:
-        # Options are all, none, strict and const
         include:
```

Comment:
we could use a

```yaml
outputs:
    matrix: ${{ steps.generate-matrix.outputs.matrix }}
```

job

which would convert the matrix in this file to

```
strategy:
  fail-fast: false
    include: ${{ fromJson(needs.generate-matrix.outputs.matrix) }}
```

however, that wouldn't test different command flags. So maybe we shouldn't do that and instead just have a single job to test all defined test targets in `svd2rust-regress`

Reply:
I do this in cross-rs, where I have a targets.toml that populate the CI automatically.

The input basically expands to a list of targets, not just a single target in a list

Reply:
Would this be for a different PR?